pupa/settings.py

```diff
@@ -72,3 +73,10 @@
 
 
 DATABASES = {'default': dj_database_url.parse(DATABASE_URL)}
+
+
+elasticsearch = None
```

Comment:
Yeah, let's do the `billy.core` style thing, but for now just this `elasticsearch` object